tests/js/fixtures/incident.js

```diff
@@ -1,13 +1,12 @@
 export function Incident(params) {
   return {
-    id: '123',
-    identifier: '123',
+    id: '321',
+    identifier: 123,
```

Comment:
Doesn't our API generally return ID values as strings?

Reply:
Yeah I'll update to be consistent